### DISCUSSION TITLE
Welcome to testcontainers-ignition Discussions!

### DISCUSSION THREAD

**Discussion by benmusson** [mussonindustrial/testcontainers-ignition#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iandoyle-dev** [comment]
Hey all, I'm Ian, a SCADA developer at Hoffman Building Technologies in Charlotte, NC.

Today, I was able to run the port mapping test.

![image](https://github.com/user-attachments/assets/3b53b18b-4887-4866-b1a3-4f06a1a4bb7e)

<testsuite name="com.mussonindustrial.testcontainers.ignition.PortMappingTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2024-08-26T18:12:42"...

Last week, I used Pyro MUI Joy to start work on a front-end theme. Also, I've had success creating an Embr Event Stream with version 0.2.0. I'm really looking forward to SSE.

Really great work! Hope I can help contribute to the conversation.

**Comment by benmusson** [comment]
Hey Ian, that's awesome! I'm glad to hear the builds are reproducible 😁 
Are you gonna be at ICC?

**Reply by iandoyle-dev** [reply]
I was able to attend last year, but not this year. This is an exciting year to attend! Will you be there?
We are pushing for Premier Integrator status this next year, so I'm organizing resources that I can share to support the community.

**Reply by benmusson** [reply]
Yup, we'll be there.
Cool, good luck with the Premier Integrator acknowledgement!
